pkgs/tools/graphics/ldgallery/viewer/default.nix

```diff
@@ -9,16 +9,13 @@ let
   sourcePkg = fetchFromGitHub {
     owner = "pacien";
     repo = "ldgallery";
-    rev = "v2.0";
-    sha256 = "1a82wy6ns1434gdba2l04crvr5waf03y02bappcxqci2cfb1cznz";
+    rev = "v2.1";
```

Comment:
Shouldn't this be derived from ldgallery-compiler.version?

Reply:
It can be if you'd rather! I tried to make mostly minimal changes to get apps to build again after the update, so I left patterns like that as they were before. I'm happy to make the change though to avoid version desync for this particular package